src/api/utils/classification.py

```diff
@@ -153,6 +153,8 @@ def correct_image_orientation(image_record: Image):
     except (AttributeError, KeyError, IndexError) as _:
         pass
 
+    # Saving the orientated image back to the image record
```

Comment:
How about renaming `correct_image_orientation` to something like `save_normalized_imagefile` and then we add any additional file editing steps here? Right now it's stripping EXIF as well as correcting image orientation, and that's a major step; there could be others.